.github/workflows/publish-package-release.yml

```diff
@@ -29,4 +29,130 @@ jobs:
   publish-release:
     runs-on: ubuntu-latest
     steps:
-      - run: echo "Publishing release for ${{inputs.package}}"
+    - run: echo "Publishing release for ${{inputs.package}}"
+    - uses: kanga333/variable-mapper@master
+      with:
+        key: "${{inputs.package}}"
+        map: |
+          {
+            "@realm/babel-plugin": {
+              "package-path": "packages/babel-plugin",
+              "package-name": "@realm/babel-plugin",
+              "version-prefix": "babel-plugin-",
+              "release-title": "Realm Babel Plugin"
```

Comment:
Admittedly, not super important, but I guess there's a precedence for naming these CAPITALIZED_SNAKE_CASE.